hunt/app/views/animals/show.html.erb

```diff
@@ -1,2 +1,9 @@
-<h1>Animals#show</h1>
-<p>Find me in app/views/animals/show.html.erb</p>
+
+<%= link_to '<button type="button" class="btn btn-dark">View all Animals</button>'.html_safe, '/' %>
+<div class="card" style="width: 18rem;">
+  <div class="card-body">
+    <h5 class=><%=@animal.name%></h5>
```

Comment:
For some additional styling, lets place the animal name in a div with the class `card-header` directly above the `card-body` div.